src/electrum_fmt_plug.c

```diff
@@ -443,7 +466,10 @@ struct fmt_main fmt_electrum = {
 		MIN_KEYS_PER_CRYPT,
 		MAX_KEYS_PER_CRYPT,
 		FMT_CASE | FMT_8_BIT | FMT_OMP | FMT_HUGE_INPUT,
-		{ NULL },
+		{
+			"kdf",
```

Comment:
It would be good to add some description what 1 or 2 means, e.g.:
`kdf [1:SHA256 2:PBKDF2-SHA512]` or `kdf (1=SHA256 2=PBKDF2-SHA512)`

Not so good: `kdf: 1 = SHA256, 2 = PBKDF2-SHA512`, because `,` is used as a separator between tunable cost parameter descriptions in `./john --list=format-all-details` and `./john --list=format-details` output. So, adding a comma inside a parameter description makes automated parsing harder.
This is also "documented" in formats.h:
```
/*
 * Descriptions (names) of tunable cost parameters for this format
 *
 * These names shouldn't contain ',', because ", " is used
 * as a separator when listing tunable cost parameters
 * in --list=format-details and --list=format-all-details.
 * The sequence of names should match the sequence of functions
 * returning tunable cost values.
 */
	char *tunable_cost_name[FMT_TUNABLE_COSTS];

```
BTW:  looks like that comment is slightly wrong, because `", "`, not `","` is used as separator for --list=format-all-details.
```
$ git grep -n list_tunable_cost_names
listconf.c:334:void list_tunable_cost_names(struct fmt_main *format, char *separator)
listconf.c:547:                 list_tunable_cost_names(format, ",");
listconf.c:682:                 list_tunable_cost_names(format, ", ");
```

Currently, we don't have a unified notation, see `./john --list=format-details|cut -f 1,13 | grep '\s.*[=:,].*$'`
I think we should decide for one variant, and aim to unify the notation across formats (and search for parameters lacking a sufficient description.